tests/components/diagnostics/test_init.py

```diff
@@ -77,19 +79,48 @@ async def test_download_diagnostics(hass, hass_client):
     """Test download diagnostics."""
     config_entry = MockConfigEntry(domain="fake_integration")
     config_entry.add_to_hass(hass)
+    hass_sys_info = await async_get_system_info(hass)
+    hass_sys_info["run_as_root"] = hass_sys_info["user"] == "root"
+    del hass_sys_info["user"]
+    custom_components = {
+        ccmp: hass.data[DATA_INTEGRATIONS][ccmp].manifest
+        for ccmp in hass.data[DATA_CUSTOM_COMPONENTS]
+    }
```

Comment:
`hass.data[DATA_CUSTOM_COMPONENTS]` is empty (`{}`) so not really needed to have this logic here as it does not do anything.